styles.css

```diff
@@ -0,0 +1,75 @@
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-link {
+  margin: 2%;
+  color: #6070ff;
+  margin-top: 10%;
+}
+
+.logo {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.icon-links {
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+  margin-left: 0;
+}
+
+.icon-links li {
+  margin-right: 5%;
+}
+
+.information {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```